lib/IDE/ModuleInterfacePrinting.cpp

```diff
@@ -497,7 +497,7 @@ void swift::ide::printSubmoduleInterface(
   // If the group name is specified, we sort them according to their source order,
   // which is the order preserved by getTopLevelDecls.
   if (GroupNames.empty()) {
-    std::sort(SwiftDecls.begin(), SwiftDecls.end(),
+    std::stable_sort(SwiftDecls.begin(), SwiftDecls.end(),
```

Comment:
@nkcsgexi Arguably we could not sort at all now, but this kept the test churn down. What do you think is the right thing to do?

Reply:
we've heard no complaint about this ordering. I think we should keep sorting them in this way.

Reply:
Okay. We should probably sort extensions too, then, but that can come later.